recipes/alm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ALM" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "a3ee1b25a897ec8bd576dd00c948657c4faefbd42f2616bf5ad2ce79a3d8cda4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jochym/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - use_external_spglib.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - eigen
+  host:
+    - python
+    - blas 1.1 openblas  # [unix]
+    - numpy
+    - boost
+    - spglib >=1.11.1.2
+    - setuptools
+  run:
+    - python
+    - blas 1.1 openblas  # [unix]
+    - {{ pin_compatible('numpy') }}
+    - spglib >=1.11.1.2
```

Comment:
This dependency is too lax and can cause the software to break if the ABI of spglib changes. 

There are two way to fix this, 
  1- In the spglib-feedstock add a `run_exports` section and then remove run dependency from here, see [conda-forge doc](https://conda-forge.org/docs/meta.html#using-conda-build-s-run-exports-feature) for more details.
  2- Use `pin_compatible` just as you did for NumPy, see [conda's doc](https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-recipe-level) for more details.

Since you are the maintainer for spglib, my suggestion is to fix the spglib recipe and not have to worry about it again for every package that depends on it.

Reply:
Opted for the first solution